File: gfg/basics/patterns/Pattern2.py
# Pattern 2:
# https://practice.geeksforgeeks.org/problems/right-triangle/1
#
# *
# * *
# * * *
# * * * *
# * * * * *

def printTriangle(N):
    for i in range(N):
        for j in range(i+1):
            print("*", end=" ")
        print("")


if __name__ == "__main__":
    printTriangle(5)

File: basics/patterns/Pattern2.java
public class Pattern2 {

    /*
    Pattern 2:
    https://practice.geeksforgeeks.org/problems/right-triangle/1

    *
    * *
    * * *
    * * * *
    * * * * *

     */

    static void printTriangle(int n) {
        // code here
        for (int i = 0; i < n; i++) {
            for (int j = 0; j <= i; j++) {
                System.out.print("* ");
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        printTriangle(6);
    }
}
